content/300-guides/200-deployment/001-deploying-prisma.mdx

```diff
@@ -19,16 +19,22 @@ Moreover, the user traffic pattern of your application is also an important fact
 
 ### Long-running process (PaaS)
 
-Your Node.js process is continuously running and handles multiple requests. Your application can be deployed to a Platform-as-a-Service like [Heroku](./deployment-guides/deploying-to-heroku), [Koyeb](./deployment-guides/deploying-to-koyeb), as a Docker container to Kubernetes, or as a Node.js process on a virtual machine.
+Your application is [traditionally deployed](./traditional-deployments) if a Node.js process is continuously running and handles multiple requests. Your application can be deployed to a Platform-as-a-Service like [Heroku](./traditional-deployments/deploying-to-heroku), [Koyeb](./traditional-deployments/deploying-to-koyeb), as a Docker container to Kubernetes, or as a Node.js process on a virtual machine.
 
 See also: [Connection management in long-running processes](/guides/performance-and-optimization/connection-management#long-running-processes)
 
 ### Serverless (FaaS)
```

Comment:
This could be better as "Serverless Functions" (and below as "Edge Functions"), as I feel that is the only way probably you actually can run applications? "Serverless" and "Edge" as itself also include "Edge Compute", "Serverless database" and so on - which are not relevant for us.